letsencrypt/README.md

```diff
@@ -150,6 +151,22 @@ Cloudflare’s newer API Tokens can be restricted to specific domains and operat
 
 However, due to some shortcomings in Cloudflare’s implementation of Tokens, Tokens created for Certbot currently require `Zone:Zone:Read` and `Zone:DNS:Edit` permissions for all zones in your account.
 
+Example credentials file using restricted API Token (recommended):
+```yaml
+dns:
+  provider: dns-clooudflare
```

Comment:
You got a little typo here which could lead to an issue if someone copies it over without double-checking the value

Reply:
Open a PR to fix that please